cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -450,8 +450,8 @@ def pretty_dependencies(self, dependencies, indent=None):
 
                 if key == "repo":
                     pretty.append(
-                        "{}{}: {}/{}".format(
-                            prefix, key, value.full_name, dependency["subfolder"]
+                        "{}{}: {}".format(
+                            prefix, key, value.full_name
```

Comment:
this is unnecessary. subfolder comes through as a separate key in the dependency dict.